apps-script/incident-response/ChatApp.gs

```diff
@@ -154,6 +157,7 @@ function concatenateAllSpaceMessages_(spaceName) {
   // concatenating all the messages.
   let userMap = new Map();
   return messages
+    .filter(message => !message.slashCommand)
```

Comment:
```suggestion
    .filter(message => message.slashCommand !== undefined)
```

Didn't catch this earlier elsewhere, not sure if it can ever be zero but better to be safe

Reply:
Done.